cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -113,6 +115,20 @@ func (command *Evaluate) Initialize(args []string) (evaluationContext *evaluate.
 	evaluationContext = &evaluate.Context{}
 	evaluationConfiguration = NewEvaluationConfiguration()
 
+	// Load the provided configuration file, if any.
```

Comment:
I think we can make this simpler such that we don't have to do all this special handling later on.

- Add a check that `command.Models` and `command.Repositories` must be empty if a configuration is loaded
- just set `command.Models` and `command.Repositories` from the configuration
- all the initialization later will work out of the box (i.e. without having to drag `providedConfiguration` through all the way)

Reply:
you need to convert the repository names back to OS specific ones, cause they will always be UNIX forward slashes